kyuubi-hive-beeline/src/test/java/org/apache/hive/beeline/ProxyAuthTest.java

```diff
@@ -40,7 +41,7 @@
  * <client-principal>
  */
 public class ProxyAuthTest {
-  private static final String driverName = "org.apache.hive.jdbc.HiveDriver";
+  private static final String driverName = "org.apache.kyuubi.jdbc.KyuubiHiveDriver";
```

Comment:
we still support `org.apache.hive.jdbc.HiveDriver` right ?

Reply:
I think we have no such guarantee, the internally used JDBC driver is not user-faced stuff, we only need to ensure that user could smoothly migrate from Hive Beeline to Kyuubi Beeline without changing the JDBC URL.